app/hallebarde/usecases/revoke_exchange_command_handler.py

```diff
@@ -0,0 +1,6 @@
+from hallebarde.infrastructure import exchange_repository, file_repository
+
+
+def handle(identifier: str) -> None:
```

Comment:
I would name this `revoke_an_exchange_by` or `revoke_an_exchange_by_its_identifier(...)`

Reply:
Also, I would make the file deletion more explicit in the name of the function